content/docs/js/Exporter configuration guide for Jaeger.md

```diff
@@ -0,0 +1,68 @@
+# Exporter configuration guide for Jaeger
+
+### Installing Jaeger locally
```

Comment:
I think this would be clearer if we said we're showing how to setup Jaeger using docker, but there are other ways (eg binary) with a link.